hasura/migrations/default/1668026788907_alter_table_public_circles_add_column_show_pending_gives/down.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."circles" drop column "show_pending_gives";
```

Comment:
glad to have a drop script!

nit: please at the trailing newline here

Reply:
it gets deleted when I click save by prettier

Reply:
done by github editor

Reply:
> it gets deleted when I click save by prettier

will look into this